<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q2</class>
 <widget class="QMainWindow" name="Q2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Q2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Calibration</string>
    </property>
    <widget class="QPushButton" name="Button_2_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 Find Corners</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_2_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>2.2 Find Intrinsic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_2_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>2.4 Find Distortion</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>19</y>
       <width>171</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>2.3 Find Extrinsic</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_2_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>2.3 Find Extrinsic</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>67</width>
        <height>22</height>
       </rect>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
